tests/test_request.py

```diff
@@ -1,27 +1,61 @@
 from connect.devops_testing import fixures
+from connect.devops_testing.request import Builder
+
+import pytest
 
 import os
 
 
-def test_request_builder_should_build_successfully_a_valid_request():
+def test_request_builder_should_fail_building_request_with_wrong_template_type():
+    with pytest.raises(ValueError):
+        Builder(request=['wrong request argument type'])
+
+
+def test_request_builder_should_use_default_asset_template():
+    request = fixures.make_request_builder() \
```

Comment:
i think is way better:

request = (
        fixures.make_request_builder()
        .from_default_asset()
        .build()
)

Reply:
> Please Add the same flake8 plugins as the openapi-client to avoid discrepances in style

They are, you already added the configuration to the project, and I haven't changed at all.

Reply:
This is only in the tests, not the actual functional code.

Reply:
Changed